### ISSUE TITLE
disable travis ci

### ISSUE THREAD

**Issue by jmvera255** [opensciencegrid/virtual-school-pilot-2020#75]
@osg-cat We need to disable Travis CI support in this repo, I have sufficient access to do it myself but wanted to check in with your first. If you'd prefer to do it the instructions are available at <https://opensciencegrid.org/technology/documentation/publish-osg-pages/#disabling-travis-ci>

**Comment by brianhlin** [comment]
One potentially sticky issue is that the school pages use a different theme (`windmill`) than what's provided in the GHA-based container deployment model (`material`). If we want to continue using this old theme, I believe we'll need some Software work. I'd recommend transitioning to the `material` theme since it's a very active, well-maintained project, whereas windmill` sees only a few commits a year.

This also applies to https://github.com/opensciencegrid/user-school-2019/issues/98

**Comment by jmvera255** [comment]
@brianhlin looks like switch to `material` is done in `mkdocs.yml`, correct? Are any other changes needed to make the switch to `material`?

**Comment by osg-cat** [comment]
Isn’t there some custom CSS for the School sites? If we switch to `material`, that will have to be checked. Can we get a preview instance up somewhere in a container or something?

**Comment by brianhlin** [comment]
Yeah, instructions for previewing pages here: https://opensciencegrid.org/technology/documentation/writing-documentation/#previewing-the-pages. In addition to the CSS, there's a bunch of HTML tables and other stuff in the markdown docs so we'd want to make sure that those still look ok after a theme switch.

I don't imagine that these old school pages need any updates so I don't think it'd hurt to put off this work until we have to create the 2021 website.

**Comment by osg-cat** [comment]
If someone else is making the changes, how can multiple people view the preview? Also, if there is time to do this now, I suggest just getting it done. The 2021 website will come soon.

**Comment by brianhlin** [comment]
The person working on the changes will need to make them publicly available on their GitHub fork. If @jmvera255 pushes her changes to the branch `travis-gha-transition`, then you could preview them with the following commands in your own local fork of the repo:

```
cd <path to your local fork>
git remote add jess git@github.com:jmvera255/virtual-school-pilot-2020.git
git checkout -b travis-gha-transition jess/travis-gha-transition
docker run --rm -p 8000:8000 -v ${PWD}:/docs squidfunk/mkdocs-material:6.1.4
```

Then look over the pages in your browser at `localhost:8000`.
